### DISCUSSION TITLE
Naming convention for new drivers

### DISCUSSION THREAD

**Discussion by Andi252** [wmbusmeters/wmbusmeters#1081] [General]
Hello,

I prepared a new driver for an Itron/Allmess heatmeter.

The meter consists of 2 parts (meter and RF module) and the meter is sold under different brands (Itron or Allmess in Germany)

The meter is called [UltraMaXX](https://www.itron.com/emea/solutions/product-catalog/cf-ultramaxx-mk)
and the RF module is called [EquaScan hMIU RF](https://www.itron.com/emea/solutions/product-catalog/equascan-hmiu-rf)

What would be the correct name for such a driver?

For now I called the driver "ultramaxx" and the file is called driver_ultramaxx.cc

"itron_ultramaxx" would be quite long and "equascan" is to common, because it is the name for the Itron RF module product line.

**Comment by Andi252** [comment]
The issue #1054 is for a heat meter from the same company but it looks like this meter will supply more values, therefor I'm even more unsure how to call my driver.

**Comment by weetmuts** [comment]
Hi! Sometimes it is a bit tricky to name the drivers. Here it is easy. The meter has the content we want so the driver name can be inspired by the meter name, ie "ultramaxx" sounds good. The rf module does not usually add any content, just a wrapper of the meter telegram. So it can be ignored and wmbusmeters does so automatically.

**Reply by Andi252** [reply]
The heat meter in #1054 is also from Itron and it is called like my RF module. Is the Manufacturer-ID different?
my telegram is 444497268504342300047A140030A52F2F_0406910100000C1422360000066D3B1315E72B007406FEFEFEFE426C1F010D7FEB0E00000006040C995B00372F2F0C7951622223
the telegram from #1054 is 
404497a65657751100047aBe0000a00406a505000004135ef80100066d150cd7ee2a0044061B050000426cdf2c0d7feB0700000006040c5c7e034e0c7938502321

**Comment by Andi252** [comment]
What do you think about "itronheat" or something like this?
